src/Symfony/Component/Config/Resource/BCResourceInterfaceChecker.php

```diff
@@ -29,7 +29,7 @@ public function supports(ResourceInterface $metadata)
 
     public function isFresh(ResourceInterface $resource, $timestamp)
     {
-        @trigger_error('Resource checking through ResourceInterface::isFresh() is deprecated since 2.8 and will be removed in 3.0', E_USER_DEPRECATED);
+        @trigger_error(sprintf('The class %s is performing resource checking through ResourceInterface::isFresh(), which is deprecated since 2.8 and will be removed in 3.0', get_class($resource)), E_USER_DEPRECATED);
```

Comment:
For consistency sake, can you wrap the class name with `"`? `The class "%s" is ...`